webapp/src/Controller/Jury/ContestController.php

```diff
@@ -944,4 +946,34 @@ private function doLock(int $contestId, bool $locked): Response
         }
         return $this->redirectToRoute('jury_contest', ['contestId' => $contestId]);
     }
+
+    #[Route(path: '/{contestId<\d+>}/public-scoreboard-data.zip', name: 'jury_public_scoreboard_data_zip')]
+    public function publicScoreboardDataZipAction(
+        int $contestId,
+        RequestStack $requestStack,
+        Request $request,
+        ScoreboardService $scoreboardService
+    ): Response {
+        /** @var Contest $contest */
+        $contest = $this->em->getRepository(Contest::class)->find($contestId);
+        if (!$contest) {
+            throw new NotFoundHttpException(sprintf('Contest with ID %s not found', $contestId));
+        }
+        return $this->dj->getScoreboardZip($request, $requestStack, $contest, $scoreboardService);
+    }
+
+    #[Route(path: '/{contestId<\d+>}/jury-scoreboard-data.zip', name: 'jury_scoreboard_data_zip')]
+    public function juryScoreboardDataZipAction(
+        int $contestId,
+        RequestStack $requestStack,
+        Request $request,
+        ScoreboardService $scoreboardService
+    ): Response {
+        /** @var Contest $contest */
+        $contest = $this->em->getRepository(Contest::class)->find($contestId);
+        if (!$contest) {
+            throw new NotFoundHttpException(sprintf('Contest with ID %s not found', $contestId));
+        }
+        return $this->dj->getScoreboardZip($request, $requestStack, $contest, $scoreboardService, true);
+    }
```

Comment:
Should this just be one action with the type being a parameter of sorts? Now the difference between the two actions is literally 1 bit.